### PR TITLE
feat(knobs): Replacing select elements with optionsKnob in multiple stories - FRONT-1253 

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

**Comment by planctus** [comment]
thanks @kalinchernev for the review, i'll check all these use cases

**Comment by planctus** [comment]
Thanks a lot @kalinchernev for the reivew:

- Some changes were lost when i recreated this pr, added them back and applied the chance also to the story-utils package

- accordion was a bad implementation, i just reverted the id knob to be a standard text field.

- The multiple contexts where the icons get removed by toggling the icon_path in language list overlay is because we use only one parameter, so it's correct.

- The cases where you said the toggler did not have effect i could not reproduce them, for the social components it might be that the icon you were resetting was the one for the hover? If that's the case i appended (icon hover) to the name's knob for making it clear. For the menu legacy is see the icon removed when we toggle the path.

File: src/ec/packages/ec-component-tag/tag.story.js [hunk 1]
```diff
@@ -35,10 +35,11 @@ const prepareTag = (data, link, aria) => {
     );
   }
   if (data.default_icon_path) {
-    data.default_icon_path = select(
+    data.default_icon_path = optionsKnob(
       'default_icon_path',
-      [defaultSprite],
+      { current: defaultSprite, 'no path': '' },
```
Comment (kalinchernev) [inline src/ec/packages/ec-component-tag/tag.story.js:40]: This one doesn't look quite right on toggling
disappearing-knob

**Comment by planctus** [comment]
Thanks again for the review, @kalinchernev, i fixed the behaviour in the tag component ;)

**Pull request merged**